src/main/java/org/jitsi/jigasi/AbstractGatewaySession.java

```diff
@@ -66,6 +66,19 @@
      */
     protected boolean gatewayMediaDropped = false;
 
+    /**
+     * Configuration property to change the resource used by focus.
+     */
+    private static final String FOCUSE_RESOURCE_PROP
+        = "org.jitsi.jigasi.FOCUS_RESOURCE";
+
+    /**
+     * Address of the focus member that has invited us to the conference.
+     * Used to identify the focus user and dispose the session when it leaves
+     * the room.
+     */
+    protected final String focusResourceAddr;
```

Comment:
can you add getter/setter for this property instead of accessing directly in other classes?

Reply:
I don't really see any simplification coming out of this move. Do you plan to use different focus resources on per session basis? The configuration option is global, which means this property is static and not bound to any specific instance. Maybe it should be a static variable in one of the bundle activators?

Reply:
I moved it there cause I needed a check moved in SoundNotificationManager.
https://github.com/jitsi/jigasi/pull/232/files#diff-d1aca1f84c48e995cd1e763c8d36828fR131
I will add the getters.